rpg.py

```diff
@@ -120,47 +129,45 @@ def act(self, players):
 
         return players
 
-    def move_input(self):
+    def move_input(self, players: List[Character]):
         move = None
         while move is None:
-            m_input = input('>')
-            move = m_input
+            move_input = input('>')
+            move = move_input
             if move == 'f':
-                pass
-            elif move == 'i':
+                break
+            if move == 'i':
                 self.use_item()
-            elif move == 's':
+            if move == 's':
                 self.get_stats()
-                self.move_input()
             else:
-                self.move_input()
-            return move
+                return self.move_input(players)
```

Comment:
This works, but still beats the point of using a while loop. Think about the looping condition.